hasher-matcher-actioner/scripts/set_threatexchange_source

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
ignorable: I went through a big bash period in my youth, and eventually regretted writing so much scripting, and wished I had written more python instead to do the same things. Bash has all sorts of exciting difficult-to-debug failure modes (if you don't `set -e` at the beginning, a single line may fail to execute, but it still executes the rest of the script).

Reply:
Nice catch. Adding now.